tests/sai_qualify/test_brcm_t0.py

```diff
@@ -0,0 +1,39 @@
+from sai_infra import *
```

Comment:
Should we have different test file for l2 and later l3 testcases? It would be easier to use pipeline parameters to create testcases bundle to save time or only run part of them.

Reply:
Yes, we are planning to do that, and this is the first step, we use different file for different test sets